crates/shared/src/ethrpc.rs

```diff
@@ -69,44 +44,29 @@ impl Display for Arguments {
             "ethrpc_max_concurrent_requests: {}",
             self.ethrpc_max_concurrent_requests
         )?;
+        writeln!(f, "ethrpc_batch_delay: {:?}", self.ethrpc_batch_delay)?;
 
         Ok(())
     }
 }
 
+impl From<&Arguments> for ethrpc::Arguments {
+    fn from(args: &Arguments) -> ethrpc::Arguments {
```

Comment:
nit: Instead of a `From` conversion, it might make sense to just have an accessor function for this:

```rust
impl Arguments {
    pub fn ethrpc(&self) -> ethrpc::Config {
        // ...
    }
}
```